docs/development/setup_development.rst

```diff
@@ -20,21 +20,36 @@ Quick Start
 The Docker based environment is suitable for developing the web application
 and updating the documentation.
 
+Ensure Git is installed on your system. Use package management tools to update your local package index. Once the update is complete, you can download and install Git:
 
 Ubuntu or Debian GNU/Linux
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
+Run the following commands to update the packages and to install from the default Git repositories:
 
 .. code:: sh
 
    sudo apt-get update
    sudo apt-get install -y make git
 
+
+Fedora and Other Linux Distributions
```

Comment:
Further down we are providing instructions for installing Docker under Ubuntu/Debian (the previous section), but they are now placed in the Fedora section.